src/main/java/invtweaks/InvTweaksObfuscation.java

```diff
@@ -226,13 +226,15 @@ public static Container getCurrentContainer() {
     // Slot members
 
     public static boolean areSameItemType(ItemStack itemStack1, ItemStack itemStack2) {
-        return itemStack1.isItemEqual(itemStack2) || (itemStack1.isItemStackDamageable() && itemStack1
-                .getItem() == itemStack2.getItem());
+        return itemStack1 != null && itemStack2 != null && (itemStack1.isItemEqual(itemStack2) || (itemStack1.isItemStackDamageable() && itemStack1
+                .getItem() == itemStack2.getItem()));
     }
 
     public static boolean areItemsStackable(ItemStack itemStack1, ItemStack itemStack2) {
-        return itemStack1 != null && itemStack2 != null && itemStack1.isItemEqual(itemStack2) &&
-                itemStack1.isStackable() &&
+         return itemStack1 != null && itemStack2 != null && itemStack1.isItemEqual(itemStack2)
+                && itemStack1.isStackable() && itemStack1 != itemStack2
+                && itemStack1.stackSize < itemStack1.getMaxStackSize()
+                && itemStack2.stackSize < itemStack2.getMaxStackSize() &&
```

Comment:
this check is add because the spigot server will sent extra package when you continue stack on the item which is max stack size. this is part of workaround for spigot server